pkg/reconciler/inmemorychannel/controller/controller.go

```diff
@@ -116,5 +118,10 @@ func NewController(
 		Handler:    controller.HandleAll(grCh),
 	})
 
+	// Setup the watch on the config map of dispatcher config
+	configStore := config.NewEventDispatcherConfigStore(logging.FromContext(ctx))
+	configStore.WatchConfigs(cmw)
+	r.eventDispatcherConfigStore = configStore
```

Comment:
IIRC, the result of the conversation from delivery WG is that this configuration map won't be watched or updated once the pod is running, right?

Reply:
Yes, on the dispatcher, but this is in the imc controller. @vaikas has an idea regarding this, but we can do it in a followup. With this PR, and in particular that code, the controller just has always the last copy of the config, but if it's updated, it doesn't take any particular action (like redeploying the dispatcher)

Reply:
Ah, I see, makes sense.

Reply:
like it